test/common.bash

```diff
@@ -134,6 +134,14 @@ run () {
 }
 
 scope () {
+    if [[ -n $ch_one_test ]]; then
+        # Ignore scope if a single test is given.
+        if [[ $ch_one_test != "$BATS_TEST_DESCRIPTION" ]]; then
```

Comment:
If ch_one_test is gobbledegook, i.e., doesn't match anything, the test suite will pass?

Should it fail if the argument doesn't match at least one test description?

Reply:
Yes, it will skip everything and pass.

I could add a counter somewhere and make sure exactly one test is matched. But, like above, I guessed it was an OK trade-off given the audience.

I could also add a sentence in the docs explaining this behavior. Opinions?

Reply:
I don't have strong feelings in any direction. Just a thought.